project1/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: rgb(255, 220, 249);
```

Comment:
If you add `color: deeppink;` to the `body` element, then you don't need to repeat it so much below (since it's the common color for almost all your elements).